doc/run-cloud-build-locally.md

```diff
@@ -0,0 +1,197 @@
+# Running `cloudbuild.yaml` Locally with Docker
+
+`cloudbuild-local` is deprecated; however you can still iterate on the
+`cloudbuild.yaml` steps by running them manually using Docker and the same
+tooling that Cloud Build uses. This document provides a lightweight, repeatable
+workflow for running the CI pipeline locally while being able to iterate
+quickly.
+
+## Prerequisites
+
+- Docker Desktop (with `docker compose`) installed and running.
+- A local JDK 17 + Maven 3.8.x for the Maven build steps.
+- `python3` (for `synthea-uploader` unit tests) and `python -m unittest`
+  available on `$PATH`.
+- Access to the workspace root (same directory that holds `cloudbuild.yaml`).
+
+## Environment setup
+
+1. Create an external Docker network named `cloudbuild`:
+
+```bash
+docker network create cloudbuild
+```
+
+2. From the repo root, export substitutions that Cloud Build normally injects:
+
+```bash
+export _TAG=local
+export _REPOSITORY=fhir-analytics
+```
+
+3. If you need to mirror the GCP `gcloud` network configuration (Docker Compose
+   services talk to each other by name), run all commands inside this directory
```

Comment:
This sentence is a little bit unclear; are you trying to say that all commands should be run from "the root of the repo" (which is fine and a good recommendation)? If yes, then what does have that to do with GCP network config and how docker compose services talk to each other?